### PR TITLE
implemented stars

### PR DESCRIPTION
**Pull request by ErnestMk1**
- [DEMO LINK](https://ErnestMk1.github.io/layout_stars/)
- [TEST REPORT LINK](https://ErnestMk1.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  display: flex;
+  box-sizing: border-box;
```
Comment (OleziO) [inline src/style.css:15]: A better practice is to place this property in a * selector and remove it from all other places.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(./images/star-active.svg);
+}
```
Comment (OleziO) [inline src/style.css:46]: Combine these styles to optimize code.

**Review by OleziO — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 1 above)
Comment (ErnestMk1) [reply src/style.css:15]: are talking about margin? Or box-sizing?

**Review by anastasiiavorobiova — APPROVED** [review]
Awesome!

**Pull request closed without merging**